geoips/compare_outputs.py

```diff
@@ -214,95 +214,64 @@ def images_match(output_product, compare_product, fuzz="5%"):
     bool
         Return True if images match, False if they differ
     """
-    out_diffimg = get_out_diff_fname(compare_product, output_product)
     exact_out_diffimg = get_out_diff_fname(
         compare_product, output_product, flag="exact_"
     )
+    from PIL import Image
+    import numpy as np
+    from pixelmatch.contrib.PIL import pixelmatch
+
+    LOG.info("**Comparing output_product vs. compare product")
+    out_img = Image.open(output_product)
+    comp_img = Image.open(compare_product)
+    diff_img = Image.new(mode="RGB", size=comp_img.size)
+    try:
+        diff_arr = np.abs(np.array(comp_img) - np.array(out_img))
+    except ValueError:
+        LOG.interactive("    ***************************************")
+        LOG.interactive("    *** BAD Images NOT match exactly, different sizes ***")
+        LOG.interactive(
+            "    ***   output_product: %s %s ***",
+            np.array(out_img).shape,
+            output_product,
+        )
+        LOG.interactive(
+            "    ***   compare_product: %s %s ***",
+            np.array(comp_img).shape,
+            compare_product,
+        )
+        LOG.interactive("    ***************************************")
+        return False
```

Comment:
This was raising an exception when sizes didn't match - updated to catch the exception, print out the output and comparisons images (and sizes), then return False.

Reply:
This looks good to me. Thanks for the update!